modules/globebrowsing/src/layer.cpp

```diff
@@ -102,114 +94,94 @@ namespace {
         "example: Earth is a special place",
         openspace::properties::Property::Visibility::Hidden
     };
+
+    struct [[codegen::Dictionary(Layer), codegen::noexhaustive()]] Parameters {
+        // The unique identifier for this layer. May not contain '.' or spaces
+        std::string identifier;
+
+        // A human-readable name for the user interface. If this is omitted, the
+        // identifier is used instead
+        std::optional<std::string> name;
+
+        // A human-readable description of the layer to be used in informational texts
+        // presented to the user
+        std::optional<std::string> description;
+
+        // [[codegen::verbatim(ColorInfo.description)]]
+        std::optional<glm::vec3> color;
+
+        // Specifies the type of layer that is to be added. If this value is not
+        // specified, the layer is a DefaultTileLayer
+        std::optional<std::string> type [[codegen::inlist("DefaultTileLayer",
+            "SingleImageTileLayer", "SizeReferenceTileLayer", "TemporalTileLayer",
+            "TileIndexTileLayer", "ByIndexTileLayer", "ByLevelTileLayer", "SolidColor")]];
+
+        // Determine whether the layer is enabled or not. If this value is not specified,
+        // the layer is disabled
+        std::optional<bool> enabled;
+
+        // Determines whether the downloaded tiles should have a padding added to the
+        // borders
+        std::optional<bool> padTiles;
+
+        struct Settings {
+            // The opacity value of the layer
+            std::optional<float> opacity [[codegen::inrange(0.0, 1.0)]];
+
+            // The gamma value that is applied to each pixel of the layer
+            std::optional<float> gamma;
+
+            // The multiplicative factor that is applied to each pixel of the layer
+            std::optional<float> multiplier;
+
+            // An additive offset that is applied to each pixel of the layer
+            std::optional<float> offset;
+        };
+        // Specifies the render settings that should be applied to this layer
+        std::optional<Settings> settings;
+
+        // This used to be   new ReferencingVerifier("globebrowsing_layeradjustment")
```

Comment:
Why isn't it now? I find this comment a little confusing

Reply:
Leftover from debugging times where i wanted to make sure that it generated the right documentation... Removed